### PR TITLE
Make sound module compile

### PR DESCRIPTION
**Pull request by lslezak**

### CODE REVIEW

File: data/sound.yml [hunk 1]
```diff
@@ -1,34 +1,28 @@
...
+    to: src/include/sound
+  - from: "sound/src/*.desktop"
+    to: src/desktop
+  - from: "sound/src/{copyfonts,alsadrivers,joystickdrivers}"
+    to: src/bin
+  - from: sound/config/sound.rnc
+    to: src/autoyast-rnc
+  - from: "sound/agents/*.{scr}"
+    to: src/scrconf
+  - from: "sound/testsuite/tests/*.{ycp,err,out}"
```
Comment (jreidinger) [inline data/sound.yml]: There is sligthly problem with this change.
1) you move only part of testsuite and it is wrong
2) tests is not part of modules sources
3) generating Makefile delete makefile inside testsuite and tet won't work

File: data/sound.yml [hunk 2]
```diff
@@ -1,34 +1,28 @@
...
+    to: src/testsuite/tests
+  - from: "sound/doc/*.{txt,html,dia,jpg}"
```
Comment (jreidinger) [inline data/sound.yml]: you can move everything with

```
- from: "sound/doc" 
  to: "/"
```

File: data/sound.yml [hunk 3]
```diff
@@ -1,34 +1,28 @@
...
+    to: doc
+  - from: "data/*.{ycp,rb}"
+    to: src/data
+  - from: "data/doc/*.txt"
```
Comment (jreidinger) [inline data/sound.yml]: it this doc really part of src? or standalone doc? Is it installed to /usr/share/YaST2/data or not?

File: data/sound.yml (hunk 2 above)
Comment (jreidinger) [inline data/sound.yml]: I am wrong, right approach is 

```
- from: "sound/doc" 
  to: "."
```

**Comment by lslezak** [comment]
I'll rebase the pull request...

**Pull request closed without merging**